apiary.apib

```diff
@@ -261,4 +261,37 @@ This route is the destination URL for password reset confirmation
                 "errors": [
                     "You need to sign in or sign up before continuing."
                 ]
-            }
\ No newline at end of file
+            }
+
+### Get Spots [GET]
```

Comment:
This part of the documentation shouldn't be after the line 223? Looks like is missing here